src/ontology/mondo-edit.obo

```diff
@@ -412720,6 +412938,13 @@ synonym: "immunodeficiency 79" EXACT [OMIM:619238]
 xref: OMIM:619238 {source="MONDO:equivalentTo"}
 is_a: MONDO:0021094 {source="OMIM:619238"} ! immunodeficiency disease
 
+[Term]
+id: MONDO:0030982
+name: sulfide:quinone oxidoreductase deficiency
+synonym: "SQORD" EXACT ABBREVIATION [OMIM:619221]
+xref: OMIM:619221 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 ! Mendelian disease
```

Comment:
inborn error of metabolism

also add axiom linking to GO:0070224

Reply:
done